src/main/java/io/mosip/iiitb/odte/App.java

```diff
@@ -0,0 +1,15 @@
+package io.mosip.iiitb.odte;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import io.mosip.iiitb.odte.config.OnDemandAppConfig;
+import io.mosip.iiitb.odte.lib.MessageBrokerWrapper;
+
+public class App {
+    public static void main(String[] args) throws Exception {
+        Injector injector = Guice.createInjector(new AppModule());
```

Comment:
Any specific reason for it to not use the sprint boot ?

Reply:
I opted not to use Spring Boot since I felt for our use case of DI and configuration bringing in Spring was a major overhead and there were no requirements of exposing any of these through web apis. Hence I decided to go with independent tool which is good at doing one thing well, as that maintains both simplicity and performance.